tdms/src/mesh_base.cpp

```diff
@@ -765,7 +765,7 @@ void crossProduct(int v1[], int v2[], int *v1crossv2){
 }
 
 
-//mex -v mesh_base.cpp ../matlabio3/matlabio.cpp -DMIOFLAG=matlab  -I../iterater -I../matlabio3 -I../matlabio
+//mex -v mesh_base.cpp ../matlabio3/matlabio.cpp -DMIOFLAG=matlab  -I../iterator -I../matlabio3 -I../matlabio
 //[v1,f1,v2,f2]=mesh_base(1,2,1,2,1,2,1,1,1) ;
```

Comment:
Can we delete these lines if they are commented out anyway?

Reply:
🤷 I would leave them in for now... but when we've refactored, documented everything, and are all a bit happier with everything, we can probably nuke all of these commented-out lines.